sql_metadb/derived_tables/finance_po_inv_transactions.sql

```diff
@@ -0,0 +1,266 @@
+    orders_invoice.po_line_additionalCost,
+    orders_invoice.po_line_discount,
+    orders_invoice.po_line_discountType,
+    orders_invoice.po_line_distribution_value,
+    orders_invoice.po_line_distribution_type,
+    orders_invoice.po_line_currency,
+    finance_orders.fiscal_year_id AS po_line_fiscal_year_id,
+    finance_orders.fiscal_year AS po_line_fiscal_year,
+    finance_orders.ledger_id AS po_line_ledger_id,
+    finance_orders.ledger_name AS po_line_ledger_name,
+    finance_orders.budget_id AS po_line_budget_id,
+    finance_orders.budget_name AS po_line_budget_name,
+    finance_orders.fund_id AS po_line_fund_id,
+    finance_orders.fund_code AS po_line_fund_code,
+    orders_invoice.po_workflowStatus,
+    transactions_po_line.transaction_id AS po_line_transaction_id,
+    transactions_po_line.transaction_encumbrance_initial_amount,
+    transactions_po_line.transaction_encumbrance_amount_expended,
+    transactions_po_line.transaction_encumbrance_order_type,
+    transactions_po_line.transaction_encumbrance_subscription,
+    orders_invoice.invoice_id,
+    orders_invoice.folio_invoice_number,
+    orders_invoice.vendor_invoice_number,
+    orders_invoice.invoice_date,
+    orders_invoice.invoice_line_id,
+    orders_invoice.invoice_line_distribution_value,
+    orders_invoice.invoice_line_distribution_type,
+    orders_invoice.invoice_line_total,
+    orders_invoice.invoice_currency,
+    finance_invoices.fiscal_year_id AS invoice_line_fiscal_year_id,
+    finance_invoices.fiscal_year AS invoice_line_fiscal_year,
+    finance_invoices.ledger_id AS invoice_line_ledger_id,
+    finance_invoices.ledger_name AS invoice_line_ledger_name,
+    finance_invoices.budget_id AS invoice_line_budget_id,
+    finance_invoices.budget_name AS invoice_line_budget_name,
+    finance_invoices.fund_id AS invoice_line_fund_id,
+    finance_invoices.fund_code AS invoice_line_fund_code,
+    orders_invoice.invoice_vendor_id,
+    orders_invoice.invoice_vendor_name,
+    orders_invoice.invoice_status,
+    orders_invoice.invoice_exchange_rate,
+    transactions_inv.transaction_id AS invoice_line_transaction_id,
+    transactions_inv.transaction_amount AS invoice_line_transaction_amount,
+    transactions_inv.transaction_currency AS invoice_line_transaction_currency
+FROM
+    orders_invoice
+    LEFT JOIN finance AS finance_orders ON finance_orders.fund_id = orders_invoice.po_line_fund_id
+    LEFT JOIN finance AS finance_invoices ON finance_invoices.fund_id = orders_invoice.invoice_line_fund_id
+    LEFT JOIN transactions AS transactions_po_line ON transactions_po_line.sourcePoLineId = orders_invoice.po_line_id
+        AND transactions_po_line.fromFundId = orders_invoice.po_line_fund_id
+    LEFT JOIN transactions AS transactions_inv ON transactions_inv.sourceInvoiceLineId = orders_invoice.invoice_line_id
+        AND transactions_inv.fromFundId = orders_invoice.invoice_line_fund_id
+    ORDER BY
+        po_number,
+        po_line_number;
```

Comment:
While sorting on disk may have its uses, we are not generally doing this.  I will remove the ORDER BY clause.